README.md

```diff
@@ -124,6 +124,13 @@ mqttui "topic"
 
 `paru -S mqttui-bin` or `yay -S mqttui-bin`
 
+#### Homebrew (Mac or Linux)
+
+```
+brew update
```

Comment:
Doesn't Homebrew does this automatically on install (when too long ago)?

Reply:
it is possible that a user has disabled auto-update

Reply:
I would assume the default.
Also I wouldn't mention `apt update` in a Debian Install Package Guide.